### DISCUSSION TITLE
Rust source file incorrectly considered a renderscript file

### DISCUSSION THREAD

**Discussion by bjorn3** [community/community#9596] [Other Feature Feedback, Questions, & Ideas]
This happens at https://cs.github.com/bjorn3/rustc_codegen_cranelift/blob/eb158de8117f5a319d309f9a3f3eb44bf1d81b5d/src/compiler_builtins.rs. The code highlighting is broken because of this. All other rust files in this repo get the correct file type inferred (except y.rs which is both a valid bash and rust source file, so that is understandable.)

**Comment by gorzell** [comment]
Ya, that is a bit tricky since the extensions match.  We will take a look and see if we can upstream a fix to the libraries we use for language detection. Thanks for giving us a heads up.
